gamemode/modules/hobo/sv_hobo.lua

```diff
@@ -1,21 +1,37 @@
-local function MakeZombieSoundsAsHobo(ply)
-    if ply:EntIndex() == 0 then return end
-    if not ply.nospamtime then
-        ply.nospamtime = CurTime() - 2
-    end
+local sounds = {
+	[1] = {
```

Comment:
Why assign these numbers explicitly? Wouldn't it be less error prone to leave them out?